jvector-examples/src/main/java/io/github/jbellis/jvector/example/AutoBenchYAML.java

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Should we tidy up the main example directory? IMO, it is getting polluted. Maybe we should add a "tools" directory or similar.